robot-core/pom.xml

```diff
@@ -148,5 +148,10 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.geneontology</groupId>
+      <artifactId>owl-diff_2.12</artifactId>
```

Comment:
This is a weird `artifactId` but I guess it's correct.

Reply:
Scala artifacts get the Scala major version appended, so that you can publish separate releases for different Scala versions, which aren't generally binary-compatible.